password/src/Data/Password/Bcrypt.hs

```diff
@@ -126,7 +127,16 @@ data Bcrypt
 -- >>> hashPassword $ mkPassword "foobar"
 -- PasswordHash {unPasswordHash = "$2b$10$..."}
 hashPassword :: MonadIO m => Password -> m (PasswordHash Bcrypt)
-hashPassword = hashPasswordWithParams 10
+hashPassword = hashPasswordWithParams defaultParams
+
+-- | Default parameters for the 'Bcrypt' algorithm.
+--
+-- >>> defaultParams
+-- 10
+--
+-- @since 3.0.2.3
```

Comment:
The next version will be `3.0.3.0`, seeing as this exports a new term from this module.
Please also bump the `password.cabal` and add this change to the `ChangeLog.md`.